ccswbs/hjckrrh
# Issue 197 - Fix heading issue and update for test

This is an updated version of #706 that fixes the heading issue and applies a patch to the Services module. This patch is necessary for the Actions utility in the test suite to be able to create terms properly. Patched files have been committed along with the patch itself.

## Test Procedure
1. Checkout branch `iss197-patch`
2. Clear cache for the new template to take effect
3. Create an event with a category
4. Go to `/events` and ensure the title reads "Upcoming Events"
5. Go to `/events/:tid` where :tid matches the category on the test event
6. Ensure "Upcoming Events" is replaced with "Events related to ______"
7. Ensure the title of the test event is an `<h2>` not an `<h3>` tag

## Automated Test
The automated test is located in ccswbs/testing#22 and only requires `ug_testing_api` to be enabled (like every other test).
